GeometrickFX/Shapes/ConcreteDomain/Rhombicosidodecahedron.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Shapes.AbstractDomain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shapes.ConcreteDomain
+{
+    public class Rhombicosidodecahedron : ShapeGlobals, IShape
+    {
+        private Dodecahedron _decahedron;
+        public Rhombicosidodecahedron(Dodecahedron decahedron, float x, float y) : base(x, y)
+        {
+            _decahedron = decahedron;
+        }
+
+        public float Area() => _decahedron.Area() * 10;
+
+        public float[] Localize() => GetPosition();
+
+        public override string ToString()
+        {
+            return $"A = a^2 * (2135) => {Area()}";
```

Comment:
magic numbers - what is 2135?

Reply:
this 2135 represents a random number to represent a very complex formula made out of many square roots, this is for result's sake... I will remove this